tests/phpunit/Mutator/DefinitionTest.php

```diff
@@ -74,4 +87,60 @@ public function valuesProvider(): iterable
             'The diff',
         ];
     }
+
+    /**
+     * @dataProvider mutatorsProvider
+     */
+    public function test_it_must_be_instantiated_with_remedies(
+        Mutator $mutator
+    ): void {
+        $this->assertNotNull(
+            $mutator->getDefinition()->getRemedies(),
+            sprintf(
+                'Definition of [%s] must provide remedies.',
+                $mutator->getName(),
+            ),
+        );
+    }
+
+    public function mutatorsProvider(): iterable
+    {
+        $mutatorFactory = SingletonContainer::getContainer()->getMutatorFactory();
+
+        $mutators = $mutatorFactory->create(array_fill_keys(
+            ProfileList::ALL_MUTATORS,
+            []
+        ), false);
+
+        foreach ($mutators as $name => $mutator) {
+            $this->assertInstanceOf(Mutator::class, $mutator);
+
+            switch ($mutator) {
+                case $mutator instanceof TrueValue:
```

Comment:
do the mutators really need to be instantiated? The definition is static so maybe passing the mutator class is enough

Reply:
Ah so this is what I forgot to enquire about in the Discord chat. I apologise. If it can be done without instantiating I’d prefer it and will give it a spin tomorrow.

Reply:
Resolved in https://github.com/infection/infection/pull/1907/commits/165b1b0298b73e0c054183df7b8df5949fb975d9